pkg/vpcmodel/output.go

```diff
@@ -237,13 +240,19 @@ func WriteToFile(content, fileName string) (string, error) {
 	return content, nil
 }
 
-// getAsteriskDetails returns the info message about how non stateful conns are marked in the output, when relevant
-func getAsteriskDetails(uc OutputUseCase, hasStatelessConn bool, outFormat OutFormat) string {
+// getAsteriskDetails returns:
+// 1. the info message about how non stateful conns are marked in the output, when relevant
+// 2. the info message about how overAproximated conns are marked in the output, when relevant
+func getAsteriskDetails(uc OutputUseCase, hasStatelessConn, hasOverApproximatedConn bool, outFormat OutFormat) string {
+	res := ""
 	if uc != SingleSubnet && (outFormat == Text || outFormat == MD || outFormat == Debug) && hasStatelessConn {
-		return asteriskDetails
+		res += statefulMessage
+	}
+	if uc != SingleSubnet && (outFormat == Text || outFormat == MD || outFormat == Debug) && hasOverApproximatedConn {
```

Comment:
```suggestion
	if hasOverApproximatedConn {
```

Reply:
fixed

Reply:
I meant to first check the negation of the common cond and return if it holds (the negation). I still think it better, אבל לא נלך לרבי על זה...